bpf/process/generic_calls.h

```diff
@@ -579,7 +579,7 @@ FUNC_INLINE int generic_retkprobe(void *ctx, struct bpf_map_def *calls, unsigned
 	enter = event_find_curr(&ppid, &walker);
 
 	e->common.op = MSG_OP_GENERIC_KPROBE;
-	e->common.flags |= MSG_COMMON_FLAG_RETURN;
+	e->common.flags = MSG_COMMON_FLAG_RETURN;
```

Comment:
not sure to follow why we want this change?

Reply:
setting flags early in the processing so in following changes the filter tail call can change it (which is executed before current flags setup code) (from commit changelog)

Reply:
hum for that specific line we need to initialize the flags for return kprobe, I guess it could have probably went to separate patch